l10n_it_account/models/account_account.py

```diff
@@ -16,3 +16,10 @@ def check_balance_sign_coherence(self):
         # Avoid check upon empty recordset to make it faster
         if groups:
             groups.check_balance_sign_coherence()
+
+    def is_valid_l10n_it(self):
+        self.ensure_one()
```

Comment:
Why not doing this at company level or account group level?

Reply:
I have separated this logic in a function in case it needs to be used elsewhere and not duplicate code.
If you consider it unnecessary I will delete it.

Reply:
I'm talking of adding this function at company level, the same as in l10n-spain

Reply:
I don't find it, please copy link to example and will change it to `res.company`.